ARTICLE-3.markdown

```diff
@@ -80,7 +80,12 @@ member consensus.
 
 The council shall meet monthly and review reports from all working groups
 charged with managing the operations of the cooperative. All members shall be
-welcome to participate in all council meetings.
+welcome to participate in all council meetings. Non-member Collaborators may
+attend the council meeting (without voting participation) if agreed upon by
+consensus of the guiding council.
+
```

Comment:
Only board members have voting rights. Board can take opinion of participating members and collaborators, but only the votes of board members will count.